src/Magento/FunctionalTestingFramework/Suite/SuiteGenerator.php

```diff
@@ -94,27 +95,34 @@ public function generateAllSuites($testManifest)
     {
         $suites = $testManifest->getSuiteConfig();
 
+        $exceptionCollector = new ExceptionCollector();
         foreach ($suites as $suiteName => $suiteContent) {
-            if (empty($suiteContent)) {
```

Comment:
I see these notices when I run generate:tests command. Are they expected?

NOTICE: Suite 'WYSIWYGDisabledSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'MagentoProductionModeOnlyTestSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'MagentoDeveloperModeOnlyTestSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'SearchEngineElasticsearchSuite' contains no tests and won't be generated.
[]
NOTICE: Suite 'InContextPaypalSuite' contains no tests and won't be generated.
[]

Reply:
Can you check again? Please add command used to see the error.

Reply:
I see this when I introduce a schema error in `AdminLoginSuccessfulTest`, to see if everything else gets generated except for this test. Also noticed that schema validation error causes test generation to fail, though I see this message `Generate Tests Command Run`. The generated folder is empty.

Reply:
Fixed